README.md

```diff
@@ -81,6 +81,19 @@ If you are using Visual Studio, to make files excluded from the package appear i
 
 Wildcards are supported for all nodes (`Content`, `None`, etc.). For example, `<None Include="Directory\**" />`.
 
+
+## Use an existing database
+
+If you want to initialize your project with objects from an existing database, you can do so with the following command:
+
+```
+sqlpackage /a:Extract /p:ExtractTarget=Flat /scs:"data source=.\SQLEXPRESS;initial catalog=Chinook;Trusted_Connection=true;encrypt=False" /tf:.\Tables
```

Comment:
I think it would be better to show placeholders here, instead of example values. This could then be followed by an example. I've seen too many people try to copy-paste examples like this and then be surprised when it doesn't work. I'd also recommend long parameters instead of short, since they're clearer.

```suggestion
sqlpackage /Action:Extract /Properties:ExtractTarget=Flat /SourceConnectionString:"<connection_string>" /TargetFile:<target_dacpac_file>
```

You could then add your existing command below, as long as it's clear that it's an example.